src/translations/de/de-componentTranslations.json

```diff
@@ -130,7 +130,9 @@
       "taskNotYetAvailable": "Noch nicht verfügbar",
       "paDescription": "Den Klang in den Wörtern identifizieren.",
       "sreDescription": "Lies die Sätze so schnell wie möglich und entscheide, ob sie wahr oder falsch sind.",
-      "swrDescription": "Die Wörter werden schnell auf dem Bildschirm erscheinen. Entscheide, ob sie echt oder erfunden sind."
+      "swrDescription": "Die Wörter werden schnell auf dem Bildschirm erscheinen. Entscheide, ob sie echt oder erfunden sind.",
+      "hostileAttributionName": "Mit Absicht oder durch Zufall?",
+      "hostileAttributionDescription": "Hör dir die Wörter an und wähle das passende Bild dazu aus."
     },
```

Comment:
It looks good but I think we have to add translations to Crowdin, or else they'll be overwritten when we pull. @digital-pro right?